### PR TITLE
feat: Release notes for Codacy Cloud May 2024 DOCS-634

### PR DESCRIPTION
**Pull request by nicklem**
Manually adding release notes for May 2024

### :eyes: Live preview
https://feat-release-notes-cloud-2024-05--docs-codacy.netlify.app/release-notes/cloud/cloud-2024-05/

### :construction: To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Review links to changelogs of updated tools
- [x] Update release date and remove `TODO` comments"

### CODE REVIEW

**Review by nicklem — COMMENTED** [review]
LGTM 👍

**Review by mrfyda — APPROVED** [review]

**Review by pedrocodacy — APPROVED** [review]
LGTM

**Review by machadoit — APPROVED** [review]
LGTM!

File: docs/release-notes/cloud/cloud-2024-05.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# Cloud May 2024
+
+These release notes are for the Codacy Cloud updates during May 2024.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   You can now [quickly create a new coding standard](../../organizations/using-coding-standards.md#creating) by activating code patterns in bulk from preset category groups. This option is also available when setting up a new organization to create a default coding standard. (ALA-824)
```
Comment (claudiacarpinteiro) [inline docs/release-notes/cloud/cloud-2024-05.md:18]: @nicklem Small tweak to solve Vale check and simplify some technical jargon.
Let me know if it makes sense. 🙏 

```suggestion
-   You can now [create a new coding standard](../../organizations/using-coding-standards.md#creating) by adjusting the sensitivity of preset category groups. This option is also available when setting up a new organization to create a default coding standard. (ALA-824)
```

Comment (nicklem) [reply docs/release-notes/cloud/cloud-2024-05.md:18]: Looks great! Thanks! 🫶

**Pull request merged**